app/controllers/orders_controller.rb

```diff
@@ -6,7 +6,7 @@ def index
 
   def create
     if current_user.nil?
-      flash[:login] = {color: 'red', message: "Please log in to check out}"}
+      flash[:login] = {color: 'red', message: "Please log in to check out"}
```

Comment:
Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.